src/UI/SlotArea.cpp

```diff
@@ -2154,7 +2154,7 @@ bool cSlotAreaArmor::CanPlaceArmorInSlot(int a_SlotNum, const cItem & a_Item)
 {
 	switch (a_SlotNum)
 	{
-		case 0:  return ItemCategory::IsHelmet    (a_Item.m_ItemType);
+		case 0:  return (ItemCategory::IsHelmet(a_Item.m_ItemType) || (a_Item.m_ItemType == E_BLOCK_PUMPKIN));
 		case 1:  return ItemCategory::IsChestPlate(a_Item.m_ItemType);
 		case 2:  return ItemCategory::IsLeggings  (a_Item.m_ItemType);
 		case 3:  return ItemCategory::IsBoots     (a_Item.m_ItemType);
```

Comment:
Why? is there another method of checking this? Also, do plugins have the ability to override this? Like for wearing a stone block?